liberty-ls/src/main/resources/DiagnosticMessages.properties

```diff
@@ -0,0 +1 @@
+INVALID_PROPERTY_VALUE=The value `{0}` is not valid for the property `{1}`.
```

Comment:
The bootstrap.properties contains properties, but the server.env contains variables. I think we need a slightly different diagnostic messages for the two files, unless we can come up with a generic term that makes sense.

For server.env: The value `{0}` is not valid for the variable `{1}`.